biz.aQute.bndlib/src/aQute/bnd/osgi/Verifier.java

```diff
@@ -84,7 +84,7 @@ public String toString() {
 
 	public final static Pattern	ReservedFileNames				= Pattern
 																		.compile(
-																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
+																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK\\\\$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
 																						+ "\\$Mft|\\$MftMirr|\\$LogFile|\\$Volume|\\$AttrDef|\\$Bitmap|\\$Boot|\\$BadClus|\\$Secure|"
```

Comment:
The quadruple-backslash was necessary because the string gets passed through the macro parser... it's some nasty logic involved with the expansion of `${@}` in Verifier.

Whereas the other expressions I think are intended to be macros? I really don't know.